### PR TITLE
Server issue

### PR DESCRIPTION
**Pull request by excellencetechnologies**

### CODE REVIEW

File: routes/request.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var request = require('request');
```
Comment (reactexcel) [inline routes/request.js]: this is not a route. its a service. place it in service/ folder

File: routes/request.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        headers: headers,
+        timeout: 10000,
+        body: JSON.stringify(body)
+    }, function (error, result, body) {
+        if (error) {
+            callback(400, error, "error");
+        } else if (result.statusCode == 500) {
+            callback(result, body, "notfound")
+        } else {
+            callback(result, body, 'successfully');
```
Comment (reactexcel) [inline routes/request.js]: dont use this strings 'successfully'   'notfound'   'error'  make constants and use that

File: server.js [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
-var path    = require("path");
+var app = express();
+var path = require("path");
 
-
-app.get('/',function(req,res){
-  res.sendFile(path.join(/var/www/html/shekhar/hellworld/routes+'/hello.html'));
-  //__dirname : It will resolve to your project folder.
+app.get('/', function (req, res) {
+    res.sendFile(path.join(/var/www / html / shekhar / hellworld / routes + '/hello.html'));
```
Comment (reactexcel) [inline server.js]: why is this hard coded?

File: routes/order.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    var secret = req.body.secret;
+    if (access_token == undefined) {
+        console.log("undefined ");
+        res.json({status: 0, msg: "undefined "});
+    } else if (access_token.length > 0) {
+        var body = ({secret: secret});
+        var headers = {APP_ID: config.APP_ID, "Authorization": access_token};
+        var url = '/order/totalorder/';
+        request_.request(body, headers, url, function (req, response) {
+            res.json({status: 1, statuscode: req.statusCode, body: response});
```
Comment (reactexcel) [inline routes/order.js]: in none of the api ur checking sucess/error etc

File: routes/request.js (hunk 1 above)
Comment (excellencetechnologies) [inline routes/request.js]: place it in in new folder named "service"

File: routes/request.js (hunk 2 above)
Comment (excellencetechnologies) [inline routes/request.js]: using constants here const err = "error";

File: server.js (hunk 3 above)
Comment (excellencetechnologies) [inline server.js]: delete that hardcoded , no need of that

File: routes/order.js (hunk 4 above)
Comment (excellencetechnologies) [inline routes/order.js]: sending status 1 if  successful otherwise send status 0

File: config/constant.js [hunk 5]
```diff
@@ -0,0 +1 @@
+constant ={'err': 'error', "success": "successfully", "notfound": "not found", "invalid": "Invalid Fields","err_status":"500", "success_status":"200", "undefine":"undefined" }
```
Comment (reactexcel) [inline config/constant.js]: define like this

var ERROR = 'error'
var SUCCESS = 'success'

constants are always defined in capital

**Pull request merged**